lib/metasploit/framework/data_service/remote/http/remote_db_export_data_service.rb

```diff
@@ -0,0 +1,13 @@
+require 'metasploit/framework/data_service/remote/http/response_data_helper'
+
+module RemoteDbExportDataService
+  include ResponseDataHelper
+
+  DB_EXPORT_API_PATH = '/api/v1/db_export'
```

Comment:
Doesn't match Goliath API spec. Resource names should be separated with a dash. Update to be `/api/v1/db-export`.